src/Chainweb/Pact/PactService.hs

```diff
@@ -497,11 +497,11 @@ execNewBlock mpAccess parent miner = do
                 <> " (parent height = " <> sshow pHeight <> ")"
                 <> " (parent hash = " <> sshow pHash <> ")"
 
-        blockGasLimit <- view psBlockGasLimit
-        let initState = BlockFill blockGasLimit mempty 0
+        blockGasLim <- view psBlockGasLimit
+        let initState = BlockFill blockGasLim mempty 0
```

Comment:
What is the reason for this change? Avoiding a name clash or just a redundant change?